src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseDetails.java

```diff
@@ -20,4 +20,12 @@
     private final Long id;
 
     private String grantSignatureBase64;
+
+    private String registryAddressLine1;
```

Comment:
So why are we not adding registry Address to caseData like solicitor and Pa instead of in caseDetails? Also wondering why you did not use A registryAddress object to hold the address details

Reply:
Can't use caseData, as the registry addresses are held in the application.yaml and not entered in by the user, so it is not stored against the case. The reason for this is so that the case can move from registry to registry without need to do if logic and update the case data.